testing/compatability/azure/deploy.ps1

```diff
@@ -0,0 +1,45 @@
+param($octopusServerThumbprint,
+$octopusServerUrl,
+$octopusServerApiKey,
+$adminPassword)
+
+$env:octopusServerThumbprint = $octopusServerThumbprint
+$env:octopusServerUrl = $octopusServerUrl
+$env:octopusServerApiKey = $octopusServerApiKey
+$env:adminPassword = $adminPassword
+
+$env:octopusServerRole = 'OsTestingTentacle'
+$env:octopusServerEnvironment = 'OsTesting'
+$env:tentacleVersion = '7.0.201'
+
+az group create --name Tentacle --location australiaeast
+
+# Windows 2012
+$env:virtualMachineName = 'Net6Tentacle-Window2012'
+$env:virtualMachineSku = '2012-datacenter-gensecond'
+$env:os = 'windows2012'
+az deployment group create --name Tentacle --resource-group Tentacle --template-file vm.bicep --parameters vm.bicepparam
```

Comment:
I got stuck trying to figure out how to pass the `bicepparam` file to `--parameters` along with overrides e.g. `virtualMachineName`, gave up and went with environment variables!! Cumbersome but it works!!